style/repos.rst

```diff
@@ -22,9 +22,16 @@ General
 Markup languages
 ----------------
 
-We use `reStructuredText <https://docutils.sourceforge.io/rst.html>`_ (RST) for most text files on GitHub because this is also the markup language used by `Sphinx <https://www.sphinx-doc.org/en/master/>`_, our primary documentation system. Having everything written using RST makes writing and editing documents a standard experience across all repositories and makes it easy to copy and paste content from one location to another.
+We use reStructuredText_ (RST) for most text files on GitHub because this is
+also the markup language used by Sphinx_, our primary documentation system.
 
-In addition, we are in the process of folling out automated style checks for any documents that use RST, so it's good to have them in RST for that reason too.
+Having everything written using RST makes writing and editing documents a
+standard experience across all repositories and makes it easy to copy and
+paste content from one location to another.
+
+In addition, we are in the process of folling out automated style checks for
```

Comment:
Isn't this a typo?
```suggestion
In addition, we are in the process of rolling out automated style checks for
```

Reply:
Thanks for spotting this. Thanks!

@mechanomi: Will Vale or some other prose linter be able to detect those?

Reply:
@amotl yes, in the future. we purposely started with the stock configuration which is very minimal. the priority is getting Vale on all projects. once that is done, we can start expanding the config. my rationale was to move slowly and (try not to) not break things. haha. so improvements in sequence, not in parallel